### DISCUSSION TITLE
PROTO_RADIOLINK missing

### DISCUSSION THREAD

**Discussion by AlbertFarguell** [pascallanger/DIY-Multiprotocol-TX-Module#685] [Q&A] [answered]
First of all, I want to thank to Pascal and the rest of the team the effort of developing such a wonder. I have compiled and uploaded several protocols based on the CC2500 + arduino pro-mini and everything works flawlesly. But when I have tried to add a new protocol, the RadioLink, I have found no description for it, when it should be expected to find it in bank 6, not existing by the way.
Where’s my mistake?
Thank you in advance,
Albert Farguell

**Comment by pascallanger** [comment, marked as answer]
You surely don't need the banks on an Arduino pro mini...
You need to modify the _Config.h file bank 1 to contain the protocols you want: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/dbfccad568ee7f827302390d88e59ea7722af9e1/Multiprotocol/_Config.h#L431
The RadioLink protocol is PROTO_RLINK : https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/dbfccad568ee7f827302390d88e59ea7722af9e1/Multiprotocol/_Config.h#L779
Pascal

**Comment by AlbertFarguell** [comment]
Hi Pascal,
Thank you for your prompt answer. I forgot to tell you that I am using a PPM_prot[] of my own, where I have included  six protocols I am interested in. The fact of not finding RadioLink in the BANK number 6, made me think I had downloaded a wrong version.  Besides, I don’t have any selector for the protocols because although I use PPM from a radio of my own (again) I have managed to communicate my software with yours. One of my hobbies is to refurbish old mythical transmitters, as Kraft Signature, Pro Line, etc. and transform them into full fledged modern radios, with telemetry, voice, etc. 
I repeat,  thank you for your great work.
